example/package.json

```diff
@@ -29,23 +29,23 @@
     ]
   },
   "dependencies": {
-    "@cumulus/api": "^1.10.3",
-    "@cumulus/common": "^1.10.3",
-    "@cumulus/deployment": "^1.10.3",
-    "@cumulus/discover-granules": "^1.10.3",
-    "@cumulus/discover-pdrs": "^1.10.3",
-    "@cumulus/hello-world": "^1.10.3",
-    "@cumulus/integration-tests": "^1.10.3",
-    "@cumulus/move-granules": "^1.10.3",
-    "@cumulus/parse-pdr": "^1.10.3",
-    "@cumulus/pdr-status-check": "^1.10.3",
-    "@cumulus/post-to-cmr": "^1.10.3",
-    "@cumulus/queue-granules": "^1.10.3",
-    "@cumulus/queue-pdrs": "^1.10.3",
-    "@cumulus/sf-sns-report": "^1.10.3",
-    "@cumulus/sync-granule": "^1.10.3",
+    "@cumulus/api": "^1.10.4",
+    "@cumulus/common": "^1.10.4",
+    "@cumulus/deployment": "^1.10.4",
+    "@cumulus/discover-granules": "^1.10.4",
+    "@cumulus/discover-pdrs": "^1.10.4",
+    "@cumulus/hello-world": "^1.10.4",
+    "@cumulus/integration-tests": "^1.10.4",
+    "@cumulus/move-granules": "^1.10.4",
+    "@cumulus/parse-pdr": "^1.10.4",
+    "@cumulus/pdr-status-check": "^1.10.4",
+    "@cumulus/post-to-cmr": "^1.10.4",
+    "@cumulus/queue-granules": "^1.10.4",
+    "@cumulus/queue-pdrs": "^1.10.4",
+    "@cumulus/sf-sns-report": "^1.10.4",
+    "@cumulus/sync-granule": "^1.10.4",
```

Comment:
Shouldn't we pin the versions everywhere?

Reply:
I was thinking of leaving them in the example like this, because it's an example to our end users who might want to do the auto upgrades. Although usually they don't and I don't feel strongly about this. If you do, I'll change it!

Reply:
It's fine for 1.10.4, I think, since it's only a kinda-backport from the unreleased version. For "real" backports we should probably pin it in the example, though.

Reply:
Hmmm although they don't get example as a package and they probably wouldn't see it unless they go to the backport branch. But yeah we could do that.